features/registry/registry.feature

```diff
@@ -59,10 +59,10 @@ Feature: Testing registry
     Given I have a project
     Given docker config for default image registry is stored to the :dockercfg_file clipboard
     Then I run the :image_mirror client command with:
-      | source_image | docker.io/library/busybox:latest=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage1:v1 |
-      | dest_image   | centos/ruby-25-centos7:latest=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage2:v1    |
-      | a            | <%= cb.dockercfg_file %>                                                                     |
-      | insecure     | true                                                                                         |
+      | source_image | quay.io/openshifttest/busybox:latest=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage1:v1        |
+      | dest_image   | quay.io/openshifttest/hello-openshift:aosqe=<%= cb.integrated_reg_ip %>/<%= project.name %>/myimage2:v1 |
```

Comment:
Is the dest_image update to quay expected ? @xiuwang

Reply:
@liangxia yes, it's expected. dest_image is a placeholder we added.
oc image mirror <source_image> <dest_image>

Reply:
Ack